starboard/shared/uwp/extended_resources_manager.cc

```diff
@@ -364,8 +372,9 @@ void ExtendedResourcesManager::CompileShadersAsynchronously() {
 
 void ExtendedResourcesManager::ReleaseExtendedResourcesInternal() {
   SB_DCHECK(thread_checker_.CalledOnValidThread());
-
+#if defined(INTERNAL_BUILD)
   Av1VideoDecoder::ClearFrameBufferPool();
```

Comment:
Not action needed, but curious that the VP9 doesn't have equivalent ClearFrameBufferPool call here